multicluster/controllers/multicluster/commonarea/resourceimport_controller.go

```diff
@@ -374,7 +374,16 @@ func getMCServiceImport(resImp *multiclusterv1alpha1.ResourceImport) *k8smcsv1al
 // in the remoteCommonArea.
 func (r *ResourceImportReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	// Ignore status update event via GenerationChangedPredicate
-	instance := predicate.GenerationChangedPredicate{}
+	generationPredicate := predicate.GenerationChangedPredicate{}
+	// Register this controller to ignore LabelIdentity kind of ResourceImport
+	labelIdentityResImportFilter := func(object client.Object) bool {
```

Comment:
Question - why we handle LabelIdentity differently from other kinds and create a separate reconciler only for it?

Reply:
The reason is that we expect LabelIdentity events to have a higher volume than the other (i.e. Service or ACNP ResourceExport), and do not want syncing of these resources to be blocked by potentially huge number of LabelIdentity ResourceExport requests. That's why LabelIdentity reconciler has dedicated workers.

Reply:
Done